system/ui/lib/application.py

```diff
@@ -39,7 +39,7 @@ class FontWeight(IntEnum):
 def _get_ui_state():
   """Safely import and return ui_state, or None if unavailable."""
   try:
-    from openpilot.system.ui.lib.ui_state import ui_state
+    from openpilot.selfdrive.ui.ui_state import ui_state
```

Comment:
system shouldn't import from selfdrive!

Reply:
I originally planned to remove it after https://github.com/commaai/openpilot/pull/35422/files, but removing it in this PR is fine as well.

Reply:
removed